Description: Add loongarch64 support 
 .
 rust-libseccomp-sys (0.2.1-1+loong64) unreleased; urgency=medium
 .
   * src/lib.rs: Add loongarch64 support.
Author: Dandan Zhang <zhangdandan@loongson.cn>

Last-Update: 2024-11-27

Index: libseccomp-sys/src/lib.rs
===================================================================
--- libseccomp-sys.orig/src/lib.rs
+++ libseccomp-sys/src/lib.rs
@@ -209,6 +209,7 @@ pub const SCMP_ARCH_X86_64: u32 = 0xc000
 pub const SCMP_ARCH_X32: u32 = 0x4000003e;
 pub const SCMP_ARCH_ARM: u32 = 0x40000028;
 pub const SCMP_ARCH_AARCH64: u32 = 0xc00000b7;
+pub const SCMP_ARCH_LOONGARCH64: u32 = 0xc0000102;
 pub const SCMP_ARCH_MIPS: u32 = 0x8;
 pub const SCMP_ARCH_MIPS64: u32 = 0x80000008;
 pub const SCMP_ARCH_MIPS64N32: u32 = 0xa0000008;
